tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "outDir": "build",
+        "lib": ["es6", "dom"],
+        "inlineSources": true,
+        "inlineSourceMap": true,
```

Comment:
`"strict": true,` will catch many more errors, though it also has more false positives

Reply:
Thanks!  I think we'll turn on strictness once we make a bit more progress